src/features/search/components/SearchListItem.web.tsx

```diff
@@ -9,8 +9,13 @@ import { Offer } from 'shared/offer/types'
 import { HorizontalOfferTile } from 'ui/components/tiles/HorizontalOfferTile'
 import { getSpacing } from 'ui/theme'
 
+export const headerPlaceholder = {} as const
+export const footerPlaceholder = {} as const
+
+type ListPlaceholder = typeof headerPlaceholder
+
 export type RowData = {
-  items: (Record<string, never> | Offer)[]
+  items: (ListPlaceholder | Offer)[]
```

Comment:
je pense que le plus explicite serait de montrer qu'on a un header et un footer

```suggestion
export type RowData = {
  items: (typeof headerPlaceholder | Offer | typeof footerPlaceholder)[]
```